src/components/mainPage/mainPage.tsx

```diff
@@ -4,7 +4,17 @@ class ProofEditor extends React.Component {
 
 	public render() {
 		return <div className={"addStyleHere"}>
-			<p>Willkommen auf unserer Startseite. Das hier sieht noch sehr nach Baustelle aus...</p>
+			<h1>Willkommen bei Diproche!</h1>
+			<p>"Diproche" steht für "Didactical proof checking". Das heißt im Wesentlichen, dass Diproche die
```

Comment:
So ein tag der mehrere Zeilen beinhaltet ist glaube ich am lesbarsten untergebracht, wenn man diese Zeile noch in eine neue packt und dann ist alles eingeschobene der reine Text.

Reply:
Das dürfte alles lesbarer machen. Da stimme ich dir zu.